gunfolds/utils/clingo.py

```diff
@@ -5,17 +5,17 @@
 import json
 from gunfolds.utils.calc_procs import get_process_count
 
-CAPSIZE = 1000
 CLINGO_LIMIT = 64
 PNUM = min(CLINGO_LIMIT, get_process_count(1))
 
 
 def run_clingo(command,
                exact=True,
                timeout=0,
-               capsize=CAPSIZE,
+               capsize=None,
```

Comment:
If we put the default value as a list, we get a warning "Default argument value is mutable" 
which I agree with since lists in python are pass-by-reference.
we can change the way I formulated it as a list, but for options like "enum" we need two arguments. the option(string) and a bound (integer)

Reply:
I wanted to have the option to run clingo without capsize. because clingo behaves different when we don't give it any parameter for capsize(i.e. don't include -n <capsize> in run line at all) Vs. when we set capsize to zero. (-n 0).